tips-tricks/TroubleshootingandTips.rst

```diff
@@ -679,20 +679,22 @@ You need to load data from a structured datastore.
 Solution
 --------
 
-There are two major Nifi processors for importing data into Kylo (i.e. into Hadoop): GetTableData and ImportSqoop.
+There are two major NiFi processors for importing data into Kylo (i.e. into Hadoop): GetTableData and ImportSqoop.
```

Comment:
Please consider this wording: "There are two major NiFi processors provided by Kylo for importing data into Hadoop." This makes it clear that Kylo is a framework that helps with data ingestion, and is not same as Hadoop HDFS.